src/Compiler/Service/ServiceParseTreeWalk.fs

```diff
@@ -388,12 +388,6 @@ module SyntaxTraversal =
                 seq {
                     match expr with
                     | SynExpr.Sequential(expr1 = expr1; expr2 = SynExpr.Sequential _ as expr2) ->
-                        // It's a nested sequential expression.
-                        // Visit it, but make defaultTraverse do nothing,
-                        // since we're going to traverse its descendants ourselves.
-                        yield dive expr expr.Range (fun expr -> visitor.VisitExpr(path, traverseSynExpr path, (fun _ -> None), expr))
```

Comment:
expr1 is "dive"'d right below this call, and expr2 is handled via the recursive invocation => cannot see any impact from ommiting dive into the expr itself.

=> LGTM.

Reply:
Does `visitor.VisitExpr` still get called on `expr` itself now though? Maybe it does and I'm just not awake yet :)